### PR TITLE
Update gmf-backgroundlayerselector example in doc

### PR DESCRIPTION
**Pull request by fredj**
Minor documentation update

### CODE REVIEW

File: contribs/gmf/src/directives/backgroundlayerselector.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ gmf.module.value('gmfBackgroundlayerselectorTemplateUrl',
  *
  *      <gmf-backgroundlayerselector
  *        gmf-backgroundlayerselector-map="::ctrl.map">
+ *        gmf-backgroundlayerselector-select="onBackgroundSelected()">
```
Comment (sbrunner) [inline contribs/gmf/src/directives/backgroundlayerselector.js]: It's not documented in `htmlAttribute` ...

Comment (fredj) [inline contribs/gmf/src/directives/backgroundlayerselector.js]: added

Comment (sbrunner) [inline contribs/gmf/src/directives/backgroundlayerselector.js]: Thanks :-)

**Comment by sbrunner** [comment]
Looks good to me

**Pull request merged**